docs/source/rest.html.md.erb

```diff
@@ -95,7 +95,7 @@ You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wik
 Type: string
 </td>
 <td markdown="1">
-If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 639-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
+If specified, restrict the search results to a specific list of comma-separated countries. You can pass two letters country codes ([ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1#Officially_assigned_code_elements)).
```

Comment:
@Nagriar Sorry but I think that's exactly what my changes do: fix the link on `ISO-639-1` and change the ISO name for countries.